scripts/install.sh

```diff
@@ -1,12 +0,0 @@
-
-cp scripts/initd.sh /etc/init.d/incus
-
-if [ ! -d "/etc/incus" ]; then
-    mkdir /etc/incus
-fi
-cp incus.conf /etc/incus/incus.conf
-cp `which incus` /usr/sbin/incus
-
-touch /var/log/incus.log
-
-
```

Comment:
Something similar to this file was helpful for people not using code deploy to install incus, do we want to keep offering another way to get this setup?